CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
+## [3.1.1] - 2017-?
+### Fixed
```

Comment:
You labeled the ticket an enhancement :-)  I think it would be good to include the brief S3 compatibility reasoning in the hopes of being maximally helpful to anyone actually using manta on it's own terms.